src/pages/stacks-blockchain/transactions.md

```diff
@@ -0,0 +1,214 @@
+
+- **Generate**: Transactions are assembled according to the encoding specification.
+- **Verify and sign**: Transactions are verified to confirm if they are well-formed. Required signatures are filled in.
+- **Broadcast**: Transactions are send to a node.
+- **Register**: A miner receives transactions and adds them to the "mempool", a holding area for all the pending transactions.
+- **Process**: Miners review the mempool and select transactions for the next blocked to be mined. Depending on the transaction type, different actions can happen during this step. For example, post-conditions could be verified for a token transfer, new tokens could be minted, or an attemp to call an existing smart contract method could be made.
+- **Confirm**: Miners successfully mine blocks with a set of transactions. The transactions inside are successfully propogated to the network.
+
+## Types
+
+The Stacks 2.0 supports a set of different transaction types:
+
+| **Type**          | **Description**                                                                                                   |
+| ----------------- | ----------------------------------------------------------------------------------------------------------------- |
+| Coinbase          | The first transaction in a new block (an entity holding several transactions). Used to register for block rewards |
+| Token transfer    | Asset transfer from a sender to a recipient                                                                       |
+| Contract creation | Contract instantiation                                                                                            |
+| Contract call     | Contract call for a public, non read-only function                                                                |
+| Poison Microblock | Punish leaders who intentionally equivocate about the microblocks they package                                    |
+
+-> A sample of each transaction type can be found in the [Stacks Blockchain API response definition for transactions](https://blockstack.github.io/stacks-blockchain-api/#operation/get_transaction_by_id).
+
+## Post-conditions
+
+Transaction post-conditions are a feature meant to limit the damage smart contract bugs can do in terms of destroying a user's assets whenever a contract is instantiated or a public method of an existing contract is executed. Post-conditions are meant to be added by the user (or by the user's wallet software) at the moment they sign a transaction.
+
+Post-conditions are intended to be used to force a transaction to abort if the transaction would cause a principal to send an asset in a way that is not to the user's liking. For example, a user may append a post-condition saying that upon successful execution, their account's STX balance should have decreased by no more than 1 STX. If this is not the case, then the transaction would abort and the account would only pay the transaction fee of processing it.
```

Comment:
Some duplication in this 2nd paragraph. I think they could be tightened into a single paragraph?

Reply:
Good point, reworded it a bit!